test_unstructured/partition/pdf_image/test_image.py

```diff
@@ -195,6 +195,31 @@ def test_partition_image_raises_with_invalid_language(
         )
 
 
+@pytest.mark.parametrize(
+    ("strategy"),
+    [
+        ("hi_res"),
+        ("ocr_only"),
+    ],
+)
+def test_partition_image_strategies_keep_languages_metadata(strategy):
+    filename = os.path.join(
+        DIRECTORY,
+        "..",
+        "..",
+        "..",
+        "example-docs",
+        "english-and-korean.png",
+    )
+    elements = image.partition_image(
+        filename=filename,
+        ocr_languages="eng+kor",
```

Comment:
Should we use the `languages` parameter for this test instead? I'm ok with either since this old format can get converted

Reply:
Yah, doesn't make a difference, but I'll change it since that's not what this test is supposed to be testing. Thanks for catching